express/manifest.py

```diff
@@ -95,21 +95,25 @@ def _validate_spec(self) -> None:
             raise InvalidManifest.create_from(e)
 
     @classmethod
-    def create(cls, *, base_path: str, run_id: str, component_id: str) -> "Manifest":
+    def create(
+        cls, *, project_name: str, base_path: str, run_id: str, component_id: str
+    ) -> "Manifest":
         """Create an empty manifest
 
         Args:
+            project_name: Name of the cloud project (e.g. "soy-audio-379412")
             base_path: The base path of the manifest
             run_id: The id of the current pipeline run
             component_id: The id of the current component being executed
         """
         specification = {
             "metadata": {
+                "project_name": project_name,
                 "base_path": base_path,
                 "run_id": run_id,
                 "component_id": component_id,
             },
-            "index": {"location": f"/index/{run_id}/{component_id}"},
+            "index": {"location": f"/custom_artifact/{run_id}/{component_id}/index"},
```

Comment:
The reason I put the names at the end is to easily find them on GCP. If a file ends with something it might be easier to find compared to the beginning, but no strong opinion.

Also, the "custom_artificat" was included here as that is the name of the of the bucket on GCP. This is something that needs to be included in the `base_path`, I assume?